package.json

```diff
@@ -70,6 +70,7 @@
     "@polymer/paper-button": "^3.0.1",
     "@polymer/paper-radio-group": "^3.0.1",
     "@polymer/paper-slider": "^3.0.1",
+    "@types/three": "^0.93.12",
```

Comment:
That's a good point. Three.js's API is unusually stable thanks to @mrdoob 's strong discipline for maintaining backwards-compatibility. But, I suspect there is a real risk here of the types falling out of date, and I also suspect that these are only updated as people notice that the API has changed.

Reply:
Related: https://github.com/mrdoob/three.js/issues/11552#issuecomment-451967473

Reply:
@mrdoob I would second that person's recommendation for another reason: `@types/three` do not currently work if we import individual modules. This would be easy to fix with "sidecar" type declarations.

But, it's also a large burden on you and your contributors to maintain those type files, so I understand if that doesn't seem like a good idea to you.